core/contracts/oracle/implementation/Store.sol

```diff
@@ -43,9 +43,15 @@ contract Store is StoreInterface, Withdrawable, Testable {
     /**
      * @notice Construct the Store contract.
      */
-    constructor(address _timerAddress) public Testable(_timerAddress) {
+    constructor(
+        FixedPoint.Unsigned memory _fixedOracleFeePerSecondPerPfc,
+        FixedPoint.Unsigned memory _weeklyDelayFeePerSecondPerPfc,
+        address _timerAddress
+    ) public Testable(_timerAddress) {
         _createExclusiveRole(uint(Roles.Owner), uint(Roles.Owner), msg.sender);
         createWithdrawRole(uint(Roles.Withdrawer), uint(Roles.Owner), msg.sender);
+        setFixedOracleFeePerSecondPerPfc(_fixedOracleFeePerSecondPerPfc);
```

Comment:
Is there any need to make sure that these are not greater than 100%?

Reply:
Similar to how `setFee...` make the checks

Reply:
This calls the public function, which does the checks.